viewing_party/party.py

```diff
@@ -1,23 +1,189 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
+#DONE
+import re
+
+
+def create_movie(movie_title, genre, rating):
+    dict = {}
+    
+    if movie_title is None:
+        return None
+    elif genre is None:
+        return None
+    elif rating is None:
+        return None
+    else:
+        dict["title"] = movie_title
+        dict["genre"] = genre
+        dict["rating"] = rating
+        return dict
+    pass
+
+#DONE
+def add_to_watched(user_data, movie):
+    emptyDict = {}
+    user_data["watched"].append(emptyDict)
+    user_data["watched"][0]["title"] = movie["title"]
+    user_data["watched"][0]["genre"] = movie["genre"]
+    user_data["watched"][0]["rating"] = movie["rating"]
+    return user_data
+    pass
+
+#DONE
+def add_to_watchlist(user_data, movie):
+    emptyDict = {}
+    user_data["watchlist"].append(emptyDict)
+    user_data["watchlist"][0]["title"] = movie["title"]
+    user_data["watchlist"][0]["genre"] = movie["genre"]
+    user_data["watchlist"][0]["rating"] = movie["rating"]
+    return user_data
+    pass

```

Comment:
`movie` is a dictionary so we don't need to create another dictionary and add the k,v pairs (essentially making a copy of the `movie` data into`user_data[watched]`.

```python
def add_to_watchlist(user_data, movie):
    user_data["watchlist"].append(movie)
    return user_data

```